src/main/webapp/src/components/entityAnalysisInfo.js

```diff
@@ -99,7 +99,7 @@ class EntityAnalysisInfo extends React.Component {
     function getTooltip(entity) {
       return '<div style="font-size:15px">' + entity.name + '<br>' 
           + '<b>Type: </b>' + entity.type + '<br>' 
-          + '<b>Salience: </b>' + entity.salience + '<br>' 
+          + '<b>Salience: </b>' + entity.salience.toFixed(2) + '<br>' 
```

Comment:
`toFixed()` returns the object of type `string`, here it doesn't matter, but it's important for another place (see another comment)